components/ScheduleEventDetails.tsx

```diff
@@ -109,13 +110,38 @@ export function ScheduleEventDetails({ route }: any) {
           disabled={submitting}
         />
       </View>
+      {
+        scheduleItem.attendees !== null &&
+        scheduleItem.attendees.length > 0 &&
+        <View style={{ display: "flex", flexDirection: "row" }}>
+          <Ionicons name="checkmark-circle" size={30} />
+          <Text style={{ paddingTop: 15 }}>{scheduleItem.total_attendees} confirmed attendees</Text>
```

Comment:
I would just use {alignSelf: "center"} on the Text element so that the text is vertically aligned with the checkmark. A little padding (maybe ~5) on the left of the Text element might look a little nicer too. Then maybe put a little padding (~10-15) on the top of its parent View.